config/locales/en.yml

```diff
@@ -53,3 +53,8 @@ en:
         call: "Call"
         show: "Show"
         strike: "Strike"
+  support:
+       array:
+         two_words_connector: ", "
+         last_word_connector: ", "
+  
```

Comment:
This will affect _all_ instances of `to_sentence` in the Tracker. Instead you want to pass these arguments to the specific calls to `to_sentence`.